### PR TITLE
adding effects to arrow buttons and download button

### PR DESCRIPTION
**Pull request by harshsbhandari**
Fixed issue #645.

### CODE REVIEW

File: page/src/styles/YearSelector.css [hunk 1]
```diff
@@ -9,3 +9,17 @@
   font-size: 5rem;
   margin: 0;
 }
+
```
Comment (scraly) [inline page/src/styles/YearSelector.css:12]: Please add also:

```
.arrowButton{
  cursor: pointer;
}
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
Please add the pointer cursor from all the arrow buttons.

Also fo these ones:
image

**Comment by harshsbhandari** [comment]
> Please add the pointer cursor from all the arrow buttons.
> 
> Also for these ones: image

Done ma'am, sorry I missed it.

**Comment by scraly** [comment]
It is still not working for eventDateDisplay's arrow buttons ☺️.

**Comment by harshsbhandari** [comment]
> It is still not working for eventDateDisplay's arrow buttons ☺️.

Hi ma'am, made some changes, please let me know if that works

**Comment by scraly** [comment]
Still not working.

here more information about these arrows buttons:
image

**Comment by harshsbhandari** [comment]
> Still not working.
> 
> here more information about these arrows buttons: image

does it works now, made changes

**Comment by scraly** [comment]
No it does not work.

To fix it, just edit the `EventDisplay.css` file:

```
.lucide-arrow-right-circle {
  cursor: pointer;
}

.lucide-arrow-left-circle {
  cursor: pointer;
}
```

**Comment by harshsbhandari** [comment]
> No it does not work.
> 
> To fix it, just edit the `EventDisplay.css` file:
> 
> ```
> .lucide-arrow-right-circle {
>   cursor: pointer;
> }
> 
> .lucide-arrow-left-circle {
>   cursor: pointer;
> }
> ```

Made changes as instructed.

**Comment by scraly** [comment]
So with my fiw now you know in which css class you need to add effects ;-)

**Comment by harshsbhandari** [comment]
> So with my fiw now you know in which css class you need to add effects ;-)

yes, does it works fine now?

**Review by scraly — CHANGES_REQUESTED** [review]
yes the fix I show you is to have the pointer cursor.
So in order to add effects, you have to add this:

```
.lucide-arrow-right-circle :hover{
  cursor: pointer;
  transition: transform 0.5s;
  transform: scale(1.07);
  -webkit-transform: scale(1.07);
  -moz-transform: scale(1.07);
  -ms-transform: scale(1.07);
  -o-transform: scale(1.07);
  -webkit-transition: transform 0.5s;
  -moz-transition: transform 0.5s;
  -ms-transition: transform 0.5s;
  -o-transition: transform 0.5s;
}

.lucide-arrow-left-circle :hover{
  cursor: pointer;
  transition: transform 0.5s;
  transform: scale(1.07);
  -webkit-transform: scale(1.07);
  -moz-transform: scale(1.07);
  -ms-transform: scale(1.07);
  -o-transform: scale(1.07);
  -webkit-transition: transform 0.5s;
  -moz-transition: transform 0.5s;
  -ms-transition: transform 0.5s;
  -o-transition: transform 0.5s;
}
```

You have also to remove "fix" you added but that did not work.

Thanks

**Comment by scraly** [comment]
@harshsbhandari  why have you closed this PR? :)

**Comment by scraly** [comment]
I created a new PRs, now the buttons have effects, thanks! :)

**Pull request closed without merging**